vunit/modelsim_interface.py

```diff
@@ -339,14 +339,16 @@ def merge_coverage(self, file_name, args=None):
         if args is None:
             args = []
 
-        vcover_cmd = [join(self._prefix, 'vcover'), 'merge'] + args + [file_name]
-
-        for coverage_file in self._coverage_files:
-            if file_exists(coverage_file):
-                vcover_cmd.append(coverage_file)
-            else:
-                LOGGER.warning("Missing coverage file: %s", coverage_file)
-
+        vcover_cmd = [join(self._prefix, 'vcover'), 'merge', '-inputs',  join(self._output_path, 'coverage_files.txt')] + args + [file_name]
```

Comment:
use coverage_files constant from the line below instead of duplicating file name creation